tests/transformations/test_lineage.py

```diff
@@ -31,7 +33,6 @@ def sqlglot_schema() -> SQLGlotSchema:
 QUERY_KNOWN_TABLE_STAR_SELECT = "SELECT * FROM table_1"
 QUERY_UNKNOWN_TABLE_STAR_SELECT = "SELECT * FROM table_unknown"
 QUERY_ANONYMOUS_SELECT = "SELECT LEN(col_varchar) FROM table_1"
-QUERY_GIBBERISH = "%&/ GIBBERISH"
```

Comment:
this can not happen anymore, as non parseable sql strings are already caught in the dataset.